### PR TITLE
creating recipe for strawberry-graphql

### PR DESCRIPTION
**Pull request by cshaley**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by cshaley** [comment]
@conda-forge/help-python ready for review

File: recipes/strawberry-graphql/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - pip check
+    - strawberry --help
+  requires:
+    - pip
+
+about:
+  home: https://strawberry.rocks/
+  summary: A library for creating GraphQL APIs
+  license: MIT
+  license_file: LICENSE
```
Comment (BastianZim) [inline recipes/strawberry-graphql/meta.yaml:47]:
```suggestion
  license_file: LICENSE
  dev_url: https://github.com/strawberry-graphql/strawberr
  doc_url: https://strawberry.rocks/docs
```

File: recipes/strawberry-graphql/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - strawberry = strawberry.cli:run
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+    - poetry
```
Comment (BastianZim) [inline recipes/strawberry-graphql/meta.yaml:23]:
```suggestion
    - poetry >=0.12
```

File: recipes/strawberry-graphql/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  entry_points:
+    - strawberry = strawberry.cli:run
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
```
Comment (BastianZim) [inline recipes/strawberry-graphql/meta.yaml:22]:
```suggestion
    - python >=3.7
```
The upper bound is set by conda-forge once we get there.

Ref: https://github.com/strawberry-graphql/strawberry/blob/c88647514cf78c17b151ee6e8e0a17afc89c6ded/pyproject.toml#L25

File: recipes/strawberry-graphql/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - poetry
+  run:
+    - backports.cached-property >=1.0.1,<2.0.0
+    - click >=7.0,<9.0
+    - graphql-core >=3.2.0,<3.3.0
+    - pygments >=2.3,<3.0
+    - python >=3.6,<4.0
```
Comment (BastianZim) [inline recipes/strawberry-graphql/meta.yaml:29]:
```suggestion
    - python >=3.7
```

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

**Comment by cshaley** [comment]
@BastianZim Changes committed and tests passed

**Review by BastianZim — APPROVED** [review]

**Pull request merged**